src/wallet/rpcwallet.cpp

```diff
@@ -4813,7 +4813,8 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
             "                           the output is being sent to a transparent address, it’s an error to include this field.\n"
             "    }, ... ]\n"
             "3. minconf               (numeric, optional, default=" + strprintf("%u", DEFAULT_NOTE_CONFIRMATIONS) + ") Only use funds confirmed at least this many times.\n"
-            "4. fee                   (numeric, optional, default=" + strprintf("%s", FormatMoney(DEFAULT_FEE)) + ") The fee amount to attach to this transaction.\n"
+            "4. fee                   (numeric, optional, default=-1) The fee amount to attach to this transaction. The default behavior\n"
```

Comment:
I don't much like using -1 for this, but I guess we have no choice until we have named RPC arguments (#4660).

Reply:
Same.